examples/bezier/package.json

```diff
@@ -4,7 +4,6 @@
     "start": "webpack-dev-server --progress --hot --open"
   },
   "dependencies": {
-    "d3-request": "^1.0.5",
```

Comment:
Aha, not even used... Is `d3-scale` used?

Reply:
oops, will do another round of sweep and address the comments.